doc/JSON_INFO.md

```diff
@@ -831,35 +831,198 @@ The format also support snippet ids like above.
+
+#### "roof"
+(Optional) The terrain of the terrain on top of this (the roof).
+
+### Common to furniture and terrain
+Some values can/must be set for terrain and furniture. They have the same meaning in each case.
+
+#### "id"
+Id of the object, this should be unique among all object of that type (all terrain or all furniture types). By convention (but technically not needed), the id should have the "f_" prefix for furniture and the "t_" prefix for terrain. This is not translated. It must not be changed later as that would break save compatibility.
+
+#### "name"
+Displayed name of the object. This will be translated.
+
+#### "flags"
+(Optional) Various additional flags, see "doc/JSON_FLAGS.md".
+
+#### "symbol"
+ASCII symbol of the object as it appears in the game. The symbol string must be exactly one character long.
+
+#### "color" or "bgcolor"
+Color of the object as it appears in the game. "color" defines the the foreground color (no background color), "bgcolor" defines a solid background color. You must use EXACTLY ONE of "color" or "bgcolor". 
+
+#### "max_volume"
+(Optional) Maximal volume that can be used to store items here.
+
+#### "examine_action"
+(Optional) The C++ function that is called when the object is examined. See "src/iexamine.h".
+
+#### "close" and "open"
+(Optional) The value should be a terrain id (inside a terrain entry) or a furniture id (in a furniture entry). If either is defined, the player can open / close the object. Opening / closing will change the object at the affected tile to the given one. For example one could have object "safe_c", which "open"s to "safe_o" and "safe_o" in turn "close"s to "safe_c". Here "safe_c" and "safe_o" are two different terrain (or furniture) types that have different properties.
+
+#### "bash"
+(Optional) Defines whether the object can be bashed and if so, what happens. See "map_bash_info".
+
+#### "deconstruct"
+(Optional) Defines whether the object can be deconstructed and if so, what the results shall be. See "map_deconstruct_info".
+
+### map_bash_info
+Defines the various things that happen when the player or something else bashes terrain or furniture.
+
+```JSON
+{
+    "str_min": 80,
+    "str_max": 180,
+    "str_min_blocked": 15,
+    "str_max_blocked": 100,
+    "str_min_supported": 15,
+    "str_max_supported": 100,
+    "str_min_roll": 15,
+    "str_max_roll": 100,
+    "sound": "crunch!",
+    "sound_vol": 2,
+    "sound_fail": "whack!",
+    "sound_fail_vol": 2,
+    "ter_set": "t_dirt",
+    "furn_set": "f_rubble",
+    "explosive": 1,
+    "collapse_radius": 2,
+    "destroy_only": true,
+    "bash_below": true,
+    "tent_centers": ["f_groundsheet", "f_fema_groundsheet", "f_skin_groundsheet"],
+    "items": "bashed_item_result_group"
+}
+```
+
+#### "str_min", "str_max", "str_min_blocked", "str_max_blocked", "str_min_supported", "str_max_supported", "str_min_roll", "str_max_roll"
```

Comment:
I don't think anyone will miss them. I don't even understand what were they supposed to do.